src/main.js

```diff
@@ -2,9 +2,10 @@
 import {main} from './program';
 import cmd from './cmd';
 import * as logger from './util/logger';
+import {listADBDevices, listADBFirefoxAPKs} from './util/adb';
```

Comment:
As it is, as soon as we load `src/main.js` we will also load `./util/adb` and everything that `adb.js` does import statically, which include `@devicefarmer/adbkit` (and that npm package will likely load a bunch of its own dependencies as well).

As I described in one of my earlier comments on the issue related to this PR, we want to defer this "module loading chain reaction" as much as possible, which means until it is really really needed.

I think that a simple trick to achieve this lazy loading could be to define `adb` as a getter in the `util` object, something like this:
 
```
...
const utils = {
  logger,
  get adb() {
    ...
  },
}
```
and inside the getter retrieve the two methods from `./util/adb` using `require(...)` instead of the static `import` statement.

Give a try and let me know if you have any question/doubts or if it didn't work 
(you can easily double-check locally that the trick is working as expected and `./util/adb.js`is being loaded lazily, e.g. by putting a `console.log(...); // eslint-disable-line` in that file, that way your console.log will tell you when it is being loaded).